doc/qbk/04_http/08_chunked_encoding.qbk

```diff
@@ -0,0 +1,147 @@
+[@https://tools.ietf.org/html/rfc7230#section-4.1.1 ['chunk-extensions]].
+The last chunk may contain additional HTTP field values in a section
+of the last chunk called a
+[@https://tools.ietf.org/html/rfc7230#section-4.1.2 ['chunk-trailer]].
+The field values are "promised" in the header as a comma delimited list
+of field names in the
+[@https://tools.ietf.org/html/rfc7230#section-4.4 [*Trailer]]
+field value. Clients indicate their willingness to accept trailers
+by including the "trailers" token in the
+[@https://tools.ietf.org/html/rfc7230#section-4.3 [*TE]]
+field value.
+
+[heading Serializing Chunks]
+
+The __serializer__ automatically applies the chunked tranfer encoding
+when a message returns `true` from
+[link beast.ref.beast__http__message.chunked.overload1 `message::chunked`].
+The boundaries between chunks emitted by the serializer are implementation
+defined. Chunk extensions and trailers are omitted. Applications which
+need precise control over the chunk boundaries, extensions, and trailers
+may use a set of helper classes which enable manual emission of message
+payloads using chunk encoding.
+
+To use these helper classes, first serialize the header portion of the
+message using the standard interface. Then prepare the buffers, chunk
+extensions, and desired trailers, and use them with these helpers:
+
+[table Chunking Helpers
+[[Name][Description]]
+[
+    [[link beast.ref.beast__http__chunk_body `chunk_body`]]
+    [
+        A buffer sequence representing a complete chunk body.
+    ]
+][
+    [[link beast.ref.beast__http__chunk_crlf `chunk_crlf`]]
+    [
+        A buffer sequence representing the CRLF (`"\r\n"`) delimiter.
+        This class is used when the caller desires to emit the
+        chunk body in two or more individual stream operations.
+    ]
+][
+    [
+        [link beast.ref.beast__http__chunk_extensions `chunk_extensions`]
+
+
+        [link beast.ref.beast__http__basic_chunk_extensions `basic_chunk_extensions`]
+    ]
+    [
+        This is a simple, allocating container which lets callers
+        easily build up a set of chunk extensions.
+    ]
+][
+    [[link beast.ref.beast__http__chunk_header `chunk_header`]]
+    [
+        A buffer sequence representing a hex-encoded chunk size,
+        followed by an optional set of chunk extensions, including
+        the terminating CRLF (`"\r\n"`) delimiter which precedes
+        the chunk body. This class is used when the caller desires
+        to emit the chunk body in two or more individual stream
+        operations.
+    ]
+][
+    [[link beast.ref.beast__http__chunk_last `chunk_last`]]
+    [
+        A buffer sequence representing a last chunk. The last chunk
+        indicates the end of the chunked message payload, and may
+        contain optional trailer fields.
+    ]
+][
+    [
+        [link beast.ref.beast__http__make_chunk `make_chunk`]
+
+        [link beast.ref.beast__http__make_chunk `make_chunk_last`]
```

Comment:
Is this supposed to point to `beast.ref.beast__http__make_chunk`?
I do see [this page](http://vinniefalco.github.io/stage/beast/bjorn/beast/ref/beast__http__make_chunk_last.html) but not sure if this was a mistake.